pyproject.toml

```diff
@@ -13,14 +13,17 @@ python = "^3.9, <3.11"
 numpy = "~1.23.5"  # also in geoh5py
 geoh5py = {version = "~0.8.0rc1", source = "pypi", allow-prereleases = true}
 dash = {version = "~2.12", optional = true}
+
 plotly = {version = "~5.13.1", optional = true}
 scipy = {version = "~1.10.1"}
+pydantic = {version = "~2.5", optional = true}
 pyqtwebengine = {version = "~5.15", optional = true}
 pyside2 = {version = "~5.15", optional = true}
 
 ## indirect dependencies, forcing them here for installation through Conda not pip
 Pillow = "*"  # from geoh5py
 h5py = "*"  # from geoh5py
+openssl = {version = ">=3.1.4, <3.2.0a0"}
```

Comment:
**question:** what does require this new dependency? is it not already an indirect dependency?